docs/nos-client/api.md

```diff
@@ -75,6 +75,28 @@ nos.getBalance({ asset: NEO, address })
 &nbsp;
 
 
+## `getLastBlock`
+---
+The `getLastBlock` function provides the last known block that was fetched by the client.  It does
+not require the user to grant permission.
+
+#### Returns
+**object** - An object representing the last known block.  Refer to verbose response body in the NEO [getBlock](http://docs.neo.org/en-us/node/cli/2.7.4/api/getblock.html) documentation for a breakdown
+of this object's structure.
+
+#### Example
+```javascript
+const nos = window.NOS.V1;
+
+nos.getLastBlock()
+  .then((block) => console.log('Last Block:', block))
+  .catch((err) => alert(`Error: ${err.message}`));
+```
+
+
+&nbsp;
```

Comment:
Do you know about an alternative for this? 

Removing these result in the docs being too overwhelming, with everything stacked super close on each other. 
This white "enter" solves the problem, but I don't like to use this in particular :P

Reply:
@DalderupMaurice I imagine we would have to just override CSS for sections or something.  I say let's leave this for now in order to get master updated, and then let's address it after.